pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1324,12 +1335,16 @@ func appNetworkDoActivateOverlayNetwork(
 	createDefaultIpsetConfiglet(vifName, netInstStatus.DnsNameToIPList,
 		EID.String())
 
+	aclArgs := types.AppNetworkACLArgs{IsMgmt: false, BridgeName: bridgeName,
+		VifName: vifName, BridgeIP: olStatus.BridgeIPAddr, AppIP: EID.String(),
+		UpLinks: netInstStatus.IfNameList}
```

Comment:
I like the explicit UpLinks field.

But don't you need to set it in all the cases where you create/modify/(delete?) the ACLs?

Reply:
For delete we dont need, as we will be deleting the iptables rules, already compiled and stored in appNetworkStatus.